rust/cubesql/cubesql/src/lib.rs

```diff
@@ -86,9 +87,33 @@ impl fmt::Display for CubeError {
     }
 }
 
-impl<T> From<cubeclient::apis::Error<T>> for CubeError {
-    fn from(v: cubeclient::apis::Error<T>) -> Self {
-        CubeError::from_error(v)
+impl From<cubeclient::apis::Error<LoadV1Error>> for CubeError {
+    fn from(v: cubeclient::apis::Error<LoadV1Error>) -> Self {
+        let message: String = match v {
+            cubeclient::apis::Error::ResponseError(e) => match e.entity {
+                None => e.content,
+                Some(LoadV1Error::UnknownValue(_)) => e.content,
+                Some(LoadV1Error::Status4XX(unwrapped)) => unwrapped.error,
+                Some(LoadV1Error::Status5XX(unwrapped)) => unwrapped.error,
+            },
+            _ => "NOPE".to_string(),
```

Comment:
/cc @ovr Is there an idiomatic way of extracting `e.content` for this condition without having to do a `match` block?